doc/userguide/rules/ja4-keywords.rst

```diff
@@ -0,0 +1,30 @@
+JA4 Keywords
+============
+
+Suricata comes with a JA4 integration (https://github.com/FoxIO-LLC/ja4). JA4,
+as part of the larger JA4+ suite of fingerprints, is used to fingerprint TLS
+clients.
+
+We might in the future consider adding raw and original order strings (JA4_r,
```

Comment:
I do not think this should be part of the doc

Reply:
Agreed. Will remove it and stick to what is implemented so far.

Reply:
Done